stories/accordion.md

```diff
@@ -1,3 +1 @@
-# TODO -- Accordion
-
-Add some information on how this component should be used.
+TODO -- Add some information on how this component should be used.
```

Comment:
These render in the Docs pages now, and the component name is already rendered as a heading above our notes, making our heading redundant, so I've tweaked all the placeholder TODO notes accordingly.